lib/widgets/message_list.dart

```diff
@@ -31,9 +31,7 @@ class MessageListTheme extends ThemeExtension<MessageListTheme> {
     dateSeparator: Colors.black,
```

Comment:
```
theme: Replace senderName & recipientHeaderText with title

Removed `senderName` & `recipientHeaderText` from
`MessageListTheme` and used `title` from `DesignVariables`
as per the design guidelines.
discussion: https://github.com/zulip/zulip-flutter/pull/1213#discussion_r1908349307

Fixes: #973
```

Because the commit message repeats the title and the actual diff, we can remove most of it.

```
theme: Replace senderName & recipientHeaderText with title

discussion: https://github.com/zulip/zulip-flutter/pull/1213#discussion_r1908349307

Fixes: #973
```

Similarly, we should link to the Figma design.